pkg/ui/frontend/src/components/NAS/Monitor/JobList.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import { withStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import ScheduleIcon from '@material-ui/icons/Schedule';
+import HighlightOffIcon from '@material-ui/icons/HighlightOff';
+import DoneIcon from '@material-ui/icons/Done';
+import { Link } from 'react-router-dom';
+import { ListItemSecondaryAction, IconButton } from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+
+import { openDeleteDialog } from '../../../actions/generalActions';
+import DeleteDialog from '../../Menu/DeleteDialog';
+
+const module = "nasMonitor";
+
+
+const styles = theme => ({
+    running: {
+        color: '#8b8ffb',
```

Comment:
These could be added to the theme definition for reuse

Reply:
Fixed.